app/controllers/howcanihelp_controller.rb

```diff
@@ -0,0 +1,29 @@
+class HowcanihelpController < ApplicationController
```

Comment:
I didn't really know how to do it. actually it is the same controller with one change:
`    filter[:is_cocreation] = true`

Can you help me refactor?